docker/build_scripts/build_utils.sh

```diff
@@ -54,14 +54,18 @@ function do_cpython_build {
     if [ -e ${prefix}/bin/python3 ]; then
         ln -s python3 ${prefix}/bin/python
     fi
-    ${prefix}/bin/python get-pip.py
+    ${prefix}/bin/python -mensurepip
```

Comment:
-m ensurepip?

Reply:
Good catch! Was this PR tested?

Reply:
why? Both `-mensurepip` and `-m ensurepip` work.

Reply:
I see. Yes, but the second one is more readable.

Reply:
changing, thanks